build_tools/cmake/build_and_test_asan.sh

```diff
@@ -170,5 +143,4 @@ ASAN_OPTIONS=detect_leaks=0 \
     --output-on-failure \
     --no-tests=error \
     --label-regex "${vulkan_label_regex}" \
-    --label-exclude "${label_exclude_regex}" \
-    --exclude-regex "${excluded_tests_regex?}"
+    --label-exclude "${label_exclude_regex}"
```

Comment:
If someone sets `VULKAN_DISABLE`, "^driver=vulkan$" is going to be in the exclusions here and we're going to have zero matching tests. Does this make `--no-tests=error` throw an error?

Reply:
Good catch, that was indeed the case. Fixed.